elide-core/src/main/java/com/yahoo/elide/core/filter/FilterPredicate.java

```diff
@@ -197,4 +193,22 @@ public void negate() {
                 throw new InvalidOperatorNegationException();
         }
     }
+
+    /**
+     * A wrapper for filter parameters, for HQL injection.
+     */
+    @AllArgsConstructor
+    public static class FilterParameter {
+        @Getter private String name;
+        @Getter private Object value;
+
+        public String getPlaceholder() {
+            return ":" + name;
+        }
+
+        public String escapeMatching() {
+            // It is unclear why replaceAll here breaks our tests
```

Comment:
Should we investigate this? We wouldn't want to accidentally introduce an injection attack.